pkg/testing/ess/serverless.go

```diff
@@ -243,6 +248,38 @@ func (srv *ServerlessClient) WaitForKibana(ctx context.Context) error {
 	return nil
 }
 
+// ResetCredentials resets the credentials for the given ESS instance
+func (srv *ServerlessClient) ResetCredentials(ctx context.Context) (CredResetResponse, error) {
+	resetURL := fmt.Sprintf("%s/api/v1/serverless/projects/%s/%s/_reset-credentials", serverlessURL, srv.projectType, srv.proj.ID)
+
+	resetHandler, err := http.NewRequestWithContext(ctx, "POST", resetURL, nil)
+	if err != nil {
+		return CredResetResponse{}, fmt.Errorf("error creating new httpRequest: %w", err)
+	}
+
+	resetHandler.Header.Set("Content-Type", "application/json")
+	resetHandler.Header.Set("Authorization", fmt.Sprintf("ApiKey %s", srv.api))
+
+	resp, err := http.DefaultClient.Do(resetHandler)
+	if err != nil {
+		return CredResetResponse{}, fmt.Errorf("error performing HTTP request: %w", err)
+	}
+	defer resp.Body.Close()
+
+	if resp.StatusCode != http.StatusOK {
+		p, _ := io.ReadAll(resp.Body)
+		return CredResetResponse{}, fmt.Errorf("Non-201 status code returned by server: %d, body: %s", resp.StatusCode, string(p))
```

Comment:
Typo? You're checking for a 200 response code, but the message says 201.

```suggestion
		return CredResetResponse{}, fmt.Errorf("Non-200 status code returned by server: %d, body: %s", resp.StatusCode, string(p))
```